values/vault-operator/vault-operator-raw.gotmpl

```diff
@@ -293,3 +322,12 @@ resources:
       - kind: ServiceAccount
         name: vault
         namespace: vault
+{{- if $v._derived.untrustedCA }}
+  - apiVersion: v1
+    kind: Secret
+    metadata:
+      name: {{ "root-ca" }}
```

Comment:
I am confused, why is this in double brackets? If it's a variable, it should be `{{ $root-ca }}`. I get the following output:

```yaml
  otomi:template:log kind: Secret
  otomi:template:log metadata:
  otomi:template:log   labels:
  otomi:template:log     app: raw
  otomi:template:log     app.kubernetes.io/instance: vault-operator-artifacts
  otomi:template:log     app.kubernetes.io/managed-by: Helm
  otomi:template:log     app.kubernetes.io/name: raw
  otomi:template:log     app.kubernetes.io/part-of: otomi
  otomi:template:log     app.kubernetes.io/version: 0.2.3
  otomi:template:log     helm.sh/chart: raw-0.2.3
  otomi:template:log   name: root-ca
  otomi:template:log type: Opaque
  ```

Reply:
Right, double `{{}}` are not necessary here. Thanks.

Reply:
why does it need a variable? is it variable? if not just set it to the yaml prop to `root-ca`...